jupyterhub/spawner.py

```diff
@@ -52,6 +52,14 @@ class Spawner(LoggingConfigurable):
     authenticator = Any()
     api_token = Unicode()
 
+    will_resume = Bool(False,
+        help="""Whether the Spawner will resume on next start
+
```

Comment:
Perhaps for clarity add:

`Default is False where the Spawner will start anew.`
`If True, an existing Spawner will resume...`